Tests/SwiftDocCTests/DocumentationService/ConvertService/ConvertServiceTests.swift

```diff
@@ -1245,14 +1248,6 @@ class ConvertServiceTests: XCTestCase {
                         "/documentation/Test-Bundle/Default-Code-Listing-Syntax",
                     ]
                 )
-            
-                try self.assertReferenceStoreContains(
-                    referenceStore: referenceStore,
-                    topicPath: "/documentation/MyKit/MyClass",
-                    source: testBundleURL.appendingPathComponent("documentation/myclass.md"),
-                    title: "doc:MyKit/MyClass",
-                    isDocumentationExtensionContent: true
-                )
```

Comment:
Was this removed intentionally? Is there a change in behavior that now makes this fail?

Reply:
No, thanks for catching this. I missed adding this back when I restored the rest of this test.

Reply:
Fixed in https://github.com/apple/swift-docc/pull/565/commits/f7363f4a7edc17c259a21ee55a977e6b5e70ceeb